monGARS/api/authentication.py

```
@@ -1,16 +1,21 @@
-from fastapi import HTTPException, Depends
+from datetime import datetime, timezone
```

Ruff (0.12.2)

`datetime.datetime` imported but unused

Remove unused import

(F401)